x/service/keeper/msg_server_add_service.go

```diff
@@ -50,12 +50,17 @@ func (k msgServer) AddService(
 				).Error(),
 			)
 		}
-		return nil, status.Error(
-			codes.FailedPrecondition,
-			types.ErrServiceAlreadyExists.Wrapf(
-				"TODO_MAINNET_MIGRATION(@red-0ne): This is an ephemeral state of the code. Once we s/AddService/UpdateService/g, add the business logic here for updates here.",
-			).Error(),
-		)
+
+		// TODO_POST_MIGRATION: This logic will be replaced once the following PR is merged:
```

Comment:
1. Can you add a test for this as well?
- owner tries to update
- non owner tries to update

2. Upate docs too (minimal)